test/algorithms/test_qft.cpp

```diff
@@ -1,40 +1,25 @@
 #include "algorithms/QFT.hpp"
+#include "dd/Benchmark.hpp"
 #include "dd/FunctionalityConstruction.hpp"
-#include "dd/Simulation.hpp"
 
 #include "gtest/gtest.h"
 #include <cmath>
 #include <iostream>
 
 class QFT : public testing::TestWithParam<std::size_t> {
 protected:
-  void TearDown() override {
-    dd->decRef(sim);
-    dd->decRef(func);
-    dd->garbageCollect(true);
-
-    // number of complex table entries after clean-up should equal initial
-    // number of entries
-    EXPECT_EQ(dd->cn.realCount(), initialComplexCount);
-    // number of available cache entries after clean-up should equal initial
-    // number of entries
-    EXPECT_EQ(dd->cn.cacheCount(), initialCacheCount);
```

Comment:
The tests here lose this sanity check. I already annotated this some time back, but this seems to have gotten lost. I believe that also applies to the Grover test file.